kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt

```diff
@@ -1,31 +1,36 @@
 package com.dailymotion.kinta
 
 object KintaEnv {
-    const val GOOGLE_PLAY_PACKAGE_NAME = "GOOGLE_PLAY_PACKAGE_NAME"
-    const val GOOGLE_CLOUD_STORAGE_BUCKET = "GOOGLE_CLOUD_STORAGE_BUCKET"
-    const val APPCENTER_ORGANIZATION = "APPCENTER_ORGANIZATION"
-    const val SLACK_WEBHOOK_URL = "SLACK_WEBHOOK_URL"
-    const val GITLAB_PERSONAL_TOKEN = "GITLAB_PERSONAL_TOKEN"
-    const val GITHUB_TOKEN = "GITHUB_TOKEN"
-    const val GITHUB_USERNAME = "GITHUB_USERNAME"
-    const val APPCENTER_TOKEN = "APPCENTER_TOKEN"
-    const val CI_BACKEND_TOKEN = "CI_BACKEND_TOKEN"
-    const val GITHUB_APP_CLIENT_ID = "GITHUB_APP_CLIENT_ID"
-    const val GITHUB_APP_CLIENT_SECRET = "GITHUB_APP_CLIENT_SECRET"
-
-    const val TRANSIFEX_USER = "TRANSIFEX_USER"
-    const val TRANSIFEX_PASSWORD = "TRANSIFEX_PASSWORD"
-    const val TRANSIFEX_PROJECT = "TRANSIFEX_PROJECT"
-
-    const val GOOGLE_PLAY_JSON = "GOOGLE_PLAY_JSON"
-    const val GOOGLE_CLOUD_STORAGE_JSON = "GOOGLE_CLOUD_STORAGE_JSON"
-    const val JIRA_USERNAME = "JIRA_USERNAME"
-    const val JIRA_PASSWORD = "JIRA_PASSWORD"
-    const val BITRISE_PERSONAL_TOKEN = "BITRISE_PERSONAL_TOKEN"
-    const val JIRA_URL = "JIRA_URL"
-
-    const val APPLE_PASSWORD = "APPLE_PASSWORD"
-    const val APPLE_USERNAME = "APPLE_USERNAME"
+
+    enum class Env {
```

Comment:
what about ?
```suggestion
    enum class Var {
```

This way, the callsite will look like `KintaEnv.get(KintaEnv.Var. APPLE_PASSWORD)`

Reply:
👍